.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: java
-install: /bin/true
-jdk:
-- oraclejdk8
+language: java
+install: /bin/true
+jdk:
+- oraclejdk8
```

Comment:
GitHub says there's a change here but the contents are the same.

Is it a line separator issue?

Reply:
I did a dos2unix on the files because some had mixed line endings. Otherwise I didn't edit the travis.yml.

Reply:
See PRs #12  and #13. These should be fairly easy to review and then comparing this PR to the updated master branch should be easier.

Reply:
Another tip on the whitespace changes. On the "Files changed" tab, click the gear, then check "Hide whitespace", then "Apply and reload". That will help a lot. The "Split" view might be useful as well.